runner/worker/worker.go

```diff
@@ -6,20 +6,30 @@ package worker
 
 import (
 	"errors"
+	"fmt"
+
 	runner "github.com/featureform/runner"
+
 	"os"
 	"strconv"
 )
 
 func CreateAndRun() error {
 	config, ok := os.LookupEnv("CONFIG")
+
 	if !ok {
 		return errors.New("CONFIG not set")
 	}
+	fmt.Printf("Config: %v\n", config)
 	name, ok := os.LookupEnv("NAME")
+
 	if !ok {
 		return errors.New("NAME not set")
 	}
+	fmt.Printf("Name: %v\n", name)
+	if err := runner.RegisterFactory(string(runner.CREATE_TRAINING_SET), runner.TrainingSetRunnerFactory); err != nil {
```

Comment:
Should this be in an init function? Also the other factories need to be registered, including COPY_TO_ONLINE. How does it work without them?